packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/utils/polygon.dart

```diff
@@ -3,20 +3,14 @@
 // found in the LICENSE file.
 
 import '../types.dart';
+import 'maps_object.dart';
```

Comment:
Isn't maps_object.dart already exported by ../types.dart? (same for other utils)

Reply:
This is `types/utils/maps_object.dart`, not `types/maps_object.dart`, and the utils aren't exported.

(Having files with identical names in the parent and child directory would not be my first choice, but I thought that since I was already doing one arguably unrelated refactoring I should avoid another side-trip.)